README.md

```diff
@@ -122,7 +134,8 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
 
 ## Examples and Tutorials
 * Cloud ([STScI](https://github.com/spacetelescope/tike_content/blob/main/content/notebooks/data-access/data-access.ipynb))
-* [Parquet](https://github.com/IPAC-SW/ipac-sp-notebooks/blob/main/aws-open-data-catalogs/wise-allwise-catalog-demo.ipynb)
+* [IRSA Cloud Access Introduction](https://irsa.ipac.caltech.edu/docs/notebooks/cloud-access-intro.html)
+* [Parquet](https://irsa.ipac.caltech.edu/docs/notebooks/wise-allwise-catalog-demo.html)
```

Comment:
Added "IRSA Cloud Access Introduction".
Redirected "Parquet" to public website instead of private repo. (Same done for some other links below.)